### PR TITLE
Put key from .txt to .config

### PR DESCRIPTION
**Pull request by gintunas**
Deleted Mood Busters old folder, added key to .config file, put .config file to .gitignore

### CODE REVIEW

**Comment by gintunas** [comment]
The file with the key will have to be sent to each other privately.

File: Mood_Busters/Key.cs [hunk 1]
```diff
@@ -16,19 +16,13 @@ public static string[] GetKeys
...
                     {
-                        //TODO: Probably decrypt keys.txt which would be encrypted??
-
-                        return (File.ReadAllLines(String.Concat(dir.ToString(), "\\keys.txt")));
+                        return new string[] { ConfigurationManager.AppSettings.Get("Key0"), ConfigurationManager.AppSettings.Get("Key1") };
                     }
                     catch (Exception)
                     {
-                        MessageBox.Show(StringConst.ErrLicenceNotFound, StringConst.ErrLicense, MessageBoxButtons.OK, MessageBoxIcon.Error);
+                        MessageBox.Show("License key not found.", "Error_license", MessageBoxButtons.OK, MessageBoxIcon.Error);
```
Comment (pauliux00) [inline Mood_Busters/Key.cs]: You should've modified the StringConstants instead of placing hardcoded strings here

File: Mood_Busters/Key.cs [hunk 2]
```diff
@@ -14,25 +14,17 @@ public static string[] GetKeys
...
-                        return (File.ReadAllLines(String.Concat(dir.ToString(), "\\keys.txt")));
-                    }
-                    catch (Exception)
-                    {
-                        MessageBox.Show(StringConst.ErrLicenceNotFound, StringConst.ErrLicense, MessageBoxButtons.OK, MessageBoxIcon.Error);
-                        return null;
-                    }
+                try
+                {
+                    return new string[] { ConfigurationManager.AppSettings.Get("Key0"), ConfigurationManager.AppSettings.Get("Key1") };
```
Comment (pauliux00) [inline Mood_Busters/Key.cs]: Shall we keep the Key class?

Comment (pauliux00) [reply Mood_Busters/Key.cs]: I guess we don't

**Comment by pauliux00** [comment]
Looks fine for the time being

File: Mood_Busters/Key.cs (hunk 2 above)
Comment (gintunas) [reply Mood_Busters/Key.cs]: No :)

File: Mood_Busters/Key.cs (hunk 1 above)
Comment (gintunas) [reply Mood_Busters/Key.cs]: I didn't hardcode them, the master branch wasn't synced before, so this happened 😃

**Pull request merged**